prowler/lib/check/compliance_models.py

```diff
@@ -91,7 +91,6 @@ class CIS_Requirement_Attribute(BaseModel):
     AdditionalInformation: str
     DefaultValue: Optional[str]
     References: str
-    DefaultValue: Optional[str]
```

Comment:
Why are you removing this? Out of curiosity.

Reply:
It is duplicated, you can see the same value in line 92